_compdemos/intro-github.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Publishing code and data on GitHub
+last_modified_at: 2021-02-09
+main_author: Lauren Wolfe
+primary_reviewers: k8hertweck
+---
+
+This tutorial is meant to act as a step-by-step guide for researchers interested in publishing code and data associated with a manuscript. While there are different ways to go about this task, this document is focused specifically on publishing to GitHub.
+
+Using GitHub requires and basic understanding of:
+  - using the command line
+  - basic git and GitHub workflow
+  - git jargon and syntax
+  
+Resources:
+  - [Git documentation](https://git-scm.com/doc)
+  - [Introduction to unix commands](https://kb.iu.edu/d/afsk)
+    - [`cd`](https://kb.iu.edu/d/afsk#cd)
+    - [`mv`](https://kb.iu.edu/d/afsk#mv)
+    - [`cp`](https://kb.iu.edu/d/afsk#cp)
+
+This tutorial will cover:
```

Comment:
Let's frame these as questions: 
- Why publish code to GitHub? (reproducibility, encouraging citations, etc)
- How do I organize code for publishing? (you've already got this!)
- Where to do I publish code? (personal, lab, FH)
- When do I publish code? (from start, when manuscript submitted, when manuscript accepted)

Reply:
Then at the bottom you can include a basic workflow!